var/spack/repos/builtin/packages/py-matplotlib/package.py

```diff
@@ -184,7 +186,7 @@ class PyMatplotlib(PythonPackage):
         depends_on("py-pyqt4@4.6:", when="backend=" + backend, type="run")
         depends_on("qt+gui", when="backend=" + backend, type="run")
     for backend in ["qt5agg", "qt5cairo"]:
-        depends_on("py-pyqt5", when="backend=" + backend, type="run")
+        depends_on("py-pyqt5@5.12:", when="backend=" + backend, type="run")
```

Comment:
This was the one requirement I couldn't find back so quickly, and I could find was this which says that `5.15.5` is the oldest version with wheels:

https://github.com/matplotlib/matplotlib/blob/v3.10.x/.github/workflows/tests.yml#L55-L57

So this may be correct if older versions do work from source, or it may need to be bumped a bit further.

Reply:
I got this from here: https://matplotlib.org/stable/install/dependencies.html#backends

Reply:
Thanks! Good as is then.